core-web/libs/portlets/dot-content-analytics/portlet/src/lib/lib.routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Route } from '@angular/router';
+
+import { DotEnterpriseLicenseResolver, DotPushPublishEnvironmentsResolver } from '@dotcms/ui';
+
+import { DotContentAnalyticsComponent } from './dot-content-analytics/dot-content-analytics.component';
+
+export const DotContentAnalyticsRoutes: Route[] = [
+    {
+        path: '',
+        component: DotContentAnalyticsComponent,
+        providers: [DotPushPublishEnvironmentsResolver, DotEnterpriseLicenseResolver],
+        resolve: {
+            pushPublishEnvironments: DotPushPublishEnvironmentsResolver,
```

Comment:
This portlet use pushPublish?

Reply:
maybe not, will be removed if is the case.